t/ui/PhantomTest.pm

```diff
@@ -149,8 +150,19 @@ sub wait_for_ajax {
     }
 }
 
-sub javascript_console_is_empty {
-    is(pp($_driver->get_log('browser')), "[]", "no errors on javascript console");
+sub javascript_console_has_no_warnings_or_errors {
```

Comment:
I would rather drop/move this class instead of adding methods on it since relay on '.' in ```@INC``` . See https://github.com/os-autoinst/openQA/pull/1406 . But of course it's not a stopper :)

Reply:
I'll leave in this PR. This kind of refactoring can be done later. Note that this is not adding a method, it just changes the method's name.

Reply:
yeah, I saw you renamed it, and that just reminded me this class needs to be moved out of t/ui